overlord/devicestate/handlers_install.go

```diff
@@ -591,6 +598,18 @@ func (m *DeviceManager) doRestartSystemToRunMode(t *state.Task, _ *tomb.Tomb) er
 		return fmt.Errorf("missing modeenv, cannot proceed")
 	}
 
+	preseeded, err := maybeApplyPreseededData(st, boot.InitramfsUbuntuSeedDir, modeEnv.RecoverySystem, boot.InstallHostWritableDir)
+	if err != nil {
+		msg := fmt.Sprintf("failed to apply preseed data: %v", err)
+		logger.Noticef(msg)
```

Comment:
Unless I'm terribly mistaken, this is equivalent:
```suggestion
		logger.Noticef("failed to apply preseed data: %v", err)
```

Reply:
Ah yes, of course, that msg was logged also on the task so I needed a temporary variable, now it's not needed. Thanks!